gym_chess/envs/chess_env.py

```diff
@@ -47,6 +47,16 @@ def _reset(self):
     def _render(self, mode='human', close=False):
         print(self.env)
 
+    def _get_legal_move_list(self):
+      a = list(enumerate(self.env.legal_moves))
```

Comment:
May I ask the reason behind enumerating self.env.legal_moves?  Also, the Move object has a built-in method called uci() that returns the move as a string, so you could try something like:

a = list(self.env.legal_moves)
return [legal_move.uci() for legal_move in a]